roles/openshift_metrics/tasks/generate_heapster_secrets.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- when: "'secret/heapster-secrets' not in metrics_secrets.stdout_lines"
+  block:
+  - name: generate heapster secret template
+    template:
+      src: secret.j2
+      dest: "{{ mktemp.stdout }}/templates/heapster_secrets.yaml"
+      force: no
+    vars:
+      name: heapster-secrets
+      labels:
+        metrics-infra: heapster
+      data:
+        heapster.allowed-users: >
+          {{ openshift_metrics_heapster_allowed_users|b64encode }}
```

Comment:
This came from the original file (`generate_heapster_certificates.yaml`), which was renamed to `_secrets.yaml`. 

But speaking from a previous experience: yes. All entries in a secret are base64 decoded at the time they are mounted to the container.